src/Wizard.js

```diff
@@ -319,6 +316,14 @@ export default class Wizard extends Webform {
     return NativePromise.reject('Page not found');
   }
 
+  goToPage(index) {
```

Comment:
I feel that having both a `setPage` and `goToPage` method may cause a little confusion going forward. Perhaps we could add an additional argument to `setPage` to toggle this behaviour instead, perhaps with a `false` default value?

Reply:
Agreed. We should try to not duplicate nearly identical functions.